examples/lab_devices/julaboF32HD.py

```diff
@@ -12,8 +12,8 @@
 
 dut = Dut('julaboF32HD.yaml')
 dut.init()
-print("ID", dut["chiller"].get_identifier())
-print("status", dut["chiller"].get_status())
+print("ID: {}".format(dut["chiller"].get_identifier()))
```

Comment:
Since formatting is touched: we could decide on using single quotes or double quotes everywhere for string keys